File: Kotlin/fibonacci.kt
fun main(){

    var x = 1
    var y = 1

    print("Enter range: ")
    val range = readLine()!!.toInt()

    for (i in 1..range){
        print("$x + ")
        val xTem = x+y
        x = y
        y = xTem
    }
}

File: CPP/fibonacci.cpp
#include <iostream>
using namespace std;

int main() {
    int i=0, x=1, y=1, n, xTem;
    
    cout << "Enter range: ";
    cin >> n;
    
    for (i; i<n; i++){
        printf("%d + ",x);
        xTem = x+y;
        x = y;
        y = xTem;
    }
}
